<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>resources/shalqam.png</normaloff>resources/shalqam.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLineEdit" name="username_lineEdit">
           <property name="font">
            <font>
             <family>Josefin Slab</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QLineEdit{
    color: #1b3815;
    line-height: 1.2;
    position: relative;
    width: 100%;
    height: 55px;
    background: #ebebeb;
    border-radius: 27px;
    padding: 0 35px 0 35px;
}</string>
           </property>
           <property name="placeholderText">
            <string>Username</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="password_lineEdit">
           <property name="font">
            <font>
             <family>Josefin Slab</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QLineEdit{
    color: #1b3815;
    line-height: 1.2;
    position: relative;
    width: 100%;
    height: 55px;
    background: #ebebeb;
    border-radius: 27px;
    padding: 0 35px 0 35px;
}</string>
           </property>
           <property name="placeholderText">
            <string>Password</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="login_pushButton">
       <property name="font">
        <font>
         <family>Josefin Slab</family>
         <pointsize>24</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
    padding: 0 20px;
    width: 100%;
    height: 50px;
    background-color: #57b846;
    border-radius: 25px;
    color: #fff;
    line-height: 1.2;
}
</string>
       </property>
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="signup_pushButton">
       <property name="font">
        <font>
         <family>Josefin Slab</family>
         <pointsize>22</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#signup_pushButton{
    padding: 0 20px;
    width: 100%;
    height: 50px;
    background-color: #57b846;
    border-radius: 25px;
    color: #fff;
    line-height: 1.2;
}</string>
       </property>
       <property name="text">
        <string>SignUp</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
